.github/workflows/release-cnd.yml

```diff
@@ -58,21 +79,29 @@ jobs:
           asset_name: ${{ steps.create-archive.outputs.archive }}
           asset_content_type: application/gzip
 
+      - name: Check if publication to docker should be done
+        id: check-docker
+        env:
+          DOCKER_USERNAME: ${{ secrets.DOCKER_REGISTRY_USERNAME }}
+        run: |
+          echo "${{ env.DOCKER_USERNAME != '' && matrix.docker }}"
+          echo "::set-output name=process::${{ env.DOCKER_USERNAME != '' && matrix.docker }}"
```

Comment:
I don't fully understand why this is necessary? If you already have `matrix.docker`, why don't you just make everything conditional on that?

If the secrets are not present, the login will fail. Wouldn't that be preferable over the actions being silently skipped?

Reply:
no need for docker publication on forks.

On other way would be to make the job ok to fail but then when executing on comit-rs, if it actually fails we won't notice.

Reply:
I also updated the commit message: 06083973

Reply:
> This is because the job fails on forks (docker login not setup) and the fork may not want to publish a docker image.

Interesting. I would expect forks to then either delete the steps, add appropriate credentials or temporarily switch `docker` to false :D
Thanks for the clarification in the commit message!